app/workers/content_block_publishing_worker.rb

```diff
@@ -0,0 +1,62 @@
+require "sidekiq/api"
```

Comment:
still need more investigation into why this wouldn't work in the object store Engine, as we want to avoid having any code in the main repo

Reply:
ok it looked like restarting the sidekiq docker container fixed the engine issue - will move the Worker into our object store engine 👍